app/models/concerns/profile_searching.rb

```diff
@@ -24,6 +24,9 @@ module ProfileSearching
     scope :external, lambda { |external = true|
       where(external: external)
     }
+    scope :internal , lambda {
+      external(false).canonical(false)
+    }
```

Comment:
This scope name is misleading IMO - we have `external` - `internal` should be the opposite. What is it exactly you want to return with this scope, user-created profiles, policy-like profiles? I think we should name it something more descriptive.

Reply:
"policy-like profiles" - what the UI shows as policies in the list. I can call it "policy_like" if it's more clear that way

Reply:
I'm fine with policy_like.